package/MDAnalysis/analysis/hydrogenbonds/wbridge_analysis.py

```diff
@@ -678,6 +678,43 @@ def analysis(current, output, u, **kwargs):
       :attr:`~WaterBridgeAnalysis.timeseries` to find the specific time point
       of a water bridge existence.
 
+   .. attribute:: results.network
+
+      Network representation of the water network.
+
+      .. versionadded:: 2.0.0
+
+   .. attribute:: network
+
+      Alias to the :attr:`results.network` attribute.
+
+      .. deprecated:: 2.0.0
+         Will be removed in MDAnalysis 3.0.0. Please use
+         :attr:`results.hbonds` instead.
```

Comment:
Sorry if I have missed something. I wonder if I can see the discussion with regard to `results.hbonds`?

Reply:
This is a typo. @RMeli please change hbonds to network.
```suggestion
         :attr:`results.network` instead.
```

Reply:
Yep, typo (or, more likely, copy-paste gone wrong...). Thanks for spotting this!